### DISCUSSION TITLE
Translation Node

### DISCUSSION THREAD

**Discussion by XiaoLong-YANG666** [Acly/krita-ai-diffusion#845] [Q&A]
I installed a translated node，https://github.com/AlekPet/ComfyUI_Custom_Nodes_AlekPet ，I want to write one of the nodes into the plugin code (Google Translate Text Node), but failed. Please help me。

**Comment by xingewh** [comment]
The issue has now been resolved：https://github.com/Acly/krita-ai-diffusion/discussions/867
